join-domain/elx/sssd/files/join.sh

```diff
@@ -54,8 +54,11 @@ function IsDiscoverable {
 # Try to join host to domain
 function JoinDomain {
 
-  local AD_ATTRIB_OPTS
-  AD_ATTRIB_OPTS=""
+  local REALM_JOIN_OPTS
```

Comment:
You're declaring a string-var, here, but then changing it to an array-var on first use (Shellcheck probably should have barked)

```suggestion
  local -a REALM_JOIN_OPTS
```

Reply:
done